src/components/apps/launch/index.js

```diff
@@ -80,7 +80,7 @@ const Launch = ({
     );
 
     const preferences = bootstrapInfo?.preferences;
-    const notify = preferences?.enableAnalysisEmailNotification || false;
```

Comment:
Updated this here to match the suggestion for the other spot that does this. Cleaner, I think.